pi-top/pi-topOS-Web-Portal
# Continue to support AppWindow classes functionalities

| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1290) |


#### Main changes
Continue to support AppWindow classes functionalities (eg: closing windows) by using the expected window title name when opening a window.

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
- Install artifacts
- Using VNC in your pi-top[4] run `pt-os-web-portal-frontend landing`
- Go to the wi-fi page and click "Let's go"
- A new window will appear with the standalone wi-fi page.
- Press the "Close" button at the bottom. The window should close.

#### Tag anyone who definitely needs to review or help
-
